app/models/vm_or_template.rb

```diff
@@ -918,14 +906,8 @@ def log_all_proxies(all_proxy_list, message)
   end
 
   def log_proxies_vm_config
-    msg = "[#{log_proxies_format_instance(self)}] on host [#{log_proxies_format_instance(host)}] "\
-    "#{ui_lookup(:table => "storages").downcase} "
-    msg += if storage
-             "[#{storage.name}-#{storage.store_type}]"
-           else
-             "No storage"
-           end
-    msg
+    msg = "[#{log_proxies_format_instance(self)}] on host [#{log_proxies_format_instance(host)}] #{ui_lookup(:table => "storages").downcase} "
+    msg << (storage ? "[#{storage.name}-#{storage.store_type}]" : "No storage")
```

Comment:
You should add parens here, mentally I parsed this as, which I'm not even sure it is not the case.

```ruby
(msg << storage) ? "[#{storage.name}-#{storage.store_type}]" : "No storage"
```